File: DomainModel/GeoPosition.cs
namespace Swk5.GeoCaching.DomainModel {
    public struct GeoPosition {
        private double latitude;
        private double longitude;

        public GeoPosition(double latitude, double longitude) {
            this.latitude = latitude;
            this.longitude = longitude;
        }

        public double Latitude {
            get { return latitude; }
            set { latitude = value; }
        }

        public double Longitude {
            get { return longitude; }
            set { longitude = value;  }
        }

        public override int GetHashCode() {
            unchecked {
                return (Longitude.GetHashCode()*397) ^ Latitude.GetHashCode();
            }
        }

        public override string ToString() {
            return string.Format("{0},{1}", Latitude, Longitude);
        }
    }
}

File: GeoCachingWebClient/src/at/wea5/geocaching/domainmodel/GeoPosition.java
package at.wea5.geocaching.domainmodel;

public class GeoPosition {
    //------------------------------------ constructor ------------------------------------

    public GeoPosition(double latitude, double longitude) {
        this.latitude = latitude;
        this.longitude = longitude;
    }
  
    //-------------------------------------- public ---------------------------------------
        
    public double getLatitude() {
        return latitude;
    }
    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }
    public double getLongitude() {
        return longitude;
    }
    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }
    
    //------------------------------------- private ---------------------------------------
    
    //-------------------------------------- members --------------------------------------
    private double latitude;
    private double longitude;
}
